scripts/lernaAliasNorthstar.js

```diff
@@ -0,0 +1,28 @@
+const lernaAlias = require('lerna-alias');
+
+// northstar packages should pull these from npm, not the repo
+const excludedPackages = ['@fluentui/date-time-utilities', '@fluentui/dom-utilities'];
```

Comment:
also should exclude `'@uifabric/utilities'`

Reply:
That should happen automatically since the name has changed